PRIVACY.md

```diff
@@ -61,8 +61,6 @@ information about users.
 
 > You do not need to create an account to use SkyLines.
 >
-> If you do create an account, you do not need to give us your email address.
```

Comment:
can you change this to just remove the "do not" from the sentence, which was included by mistake?

Reply:
yep